README.md

```diff
@@ -92,21 +97,80 @@ Here is a handy table of configurable properties and their default values in whi
 
 #### CellConvertible 
 This protocol extends CellModel with associated type and thus can provide a default `cellClass` and `reuseIdentifier` value based on type's name.    
-It's espeatially handy when you declare your cell in XIB, becouse all you need to do is define its associated type and CellKit will provide the rest. Take a look at `XIB cell` example.
+It's espeatially handy when you declare your cell in XIB, becouse all you need to do is define its associated type and CellKit will provide the rest.  
```

Comment:
```suggestion
It's espetially handy when you declare your cell in XIB, because all you need to do is to define its associated type and CellKit will provide the rest.  
```